server_config.cfg

```diff
@@ -526,6 +561,17 @@ el_car_class_restriction=a
 el_car_restriction=
 el_weather=
 
+#pinehills raceway rally circuit
+el_add=mixed3_r3
+el_gamemode=racing
+el_laps=4
+el_bots=
+el_car_reset_disabled=0
+el_wrong_way_limiter_disabled=1
+el_car_class_restriction=a
+el_car_restriction=
+el_weather=
```

Comment:
duplicate code

Reply:
not duplicate. this adds the track going forwards, the other is the same track going backwards